deckOfCard/deckOfCards/Deck.java

```diff
@@ -0,0 +1,48 @@
+package deckOfCards;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Deck {
+    private List<Card> cardDeck;
+    private String[] str = {"3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K", "A", "2"};
+
+    public Deck() {
+        this.cardDeck = new ArrayList<Card>();
+        for (int i = 0; i < str.length; i++) {
+            for (Type type : Type.values()) {
+                cardDeck.add(new Card(str[i], type));
+            }
+        }
+    }
+
+    public List<Card> getCardDeck() {
+        return cardDeck;
+    }
+
+    @Override
+    public String toString() {
+        return "CardDeck of " + cardDeck;
+    }
+
+    //bad shuffle
+    public void shuffle() {
+        for (int i = 0; i < cardDeck.size(); i++) {
+            int index = (int) (Math.random() * cardDeck.size());
+            Card temp = cardDeck.get(i);
+            cardDeck.set(i, cardDeck.get(index));
+            cardDeck.set(index, temp);
+        }
+    }
```

Comment:
math proof:
for the 1st card: 1/52 
for the 2nd card: 51/52*1/(52-1)  = 1/52
...